docs/external-template-creation.md

```diff
@@ -108,6 +108,117 @@ Once you have the story JSON, several code changes are needed to add it to the l
 4. Verify in your WP environment that the new template is visible in the editor's "Explore Templates" section.
 5. Create a single pull request with all of the changes in steps 1-3.
 
+### Adding SVGs to the codebase as shapes
+
+First and foremost, shapes aren't complete svgs, just a single normalized path (ccordinates should be defined in a 0->1 space). To accomodate for this, there are a few formatting steps that must be taken in an svg editing software (like illustrator), and a few steps within the codebase to get a propper working path in the editor.
+
+#### Getting your svg to be a single path
+
+Okay so since we must have a single path, we can't utilize any other aspects of the svg spec. This means all shapes must be converted to a single path in your svg editing software of your choosing. For the purposes here, we will outline the steps in illustrator because it's what we've used thus far, but these svg operations should be applicable to other programs as well.
```

Comment:
```suggestion
Since we must have a single path, we can't utilize any other aspects of the svg spec. This means all shapes must be converted to a single path in the svg editing software of your choosing. For the purposes here, we will outline the steps in illustrator because it's what we've used thus far, but these svg operations should be applicable to other programs as well.
```